docs/FreeBSD.md

```diff
@@ -0,0 +1,83 @@
+In order to boot FreeBSD via PXE and use unattended install, you need the following:
+1. Extracted FreeBSD installation ISO exported via NFS.
+
+2. Relevant entry in DHCP server config:
+  `option root-path "/home/pkubaj/tftpboot/images/freebsd";`
+  `/home/pkubaj/tftpboot/images/freebsd` is the path to the extracted ISO.
```

Comment:
@pkubaj `/home/pkubaj` should be changed to some common directory. We are aiming to dockerize it, so it could be for example `/srv/tftp/images...` in the container or something similar. Any chance to run it in container?

Reply:
You can run it in a container as long as you provide the necessary files - the extracted ISO with a modified base set.